src/roswire/ros2/package.py

```diff
@@ -50,3 +179,30 @@ def _determine_paths(cls, app_instance: "AppInstance") -> List[str]:
             for (package, prefix) in package_to_prefix.items()
         ]
         return paths
+
+    def __init__(
+            self, packages: Iterable[ROS2Package], paths: Iterable[str]
```

Comment:
Excessive indentation. Readability could be improved by using a separate line for each argument.